docs/reference/sql/http-api.md

```diff
@@ -0,0 +1,39 @@
+# HTTP API
+
+To submit a SQL statement to the GreptimeDB server via HTTP API, use the following format:
+
+```shell
+curl -X POST \
+  -H 'Authorization: Basic {{authentication}}' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d 'sql={{SQL-statement}}' \
+http://{{API-host}}/v1/sql?db={{db-name}}
+```
+
+#### Method
+
+Use the POST method to submit the SQL statement to the server.
+
+#### Header
+
+* `Authorization`: The credential. Required if you are using GreptimeDB with authorized users. Please refer to [authentication](/user-guide/clients.md#HTTP-API).
```

Comment:
> using GreptimeDB with authorized users

can be misleading with *started greptime with particular operation system user*. How about *with authentication enabled`